### ISSUE TITLE
Transfer Window/Flight Planning

### ISSUE THREAD

**Issue by Damien212** [mockingbirdnest/Principia#1918]
Hi,

When using Principia in RSS, is there an accurate way of calculating transfer windows, either with porkchop plots or another method?

TWP seems to be generally inaccurate (not taking n-body effects/inclinations/precesssion etc into account maybe?), and KSPTOT has issues, even after pulling the bodies file from KSP due to the rotation Principia performs when giving bodies axial tilt.

The above problem is discussed in the KSPTOT forum thread here: [https://forum.kerbalspaceprogram.com/index.php?/topic/33568-winmac-ksp-trajectory-optimization-tool-v1510-new-mission-architect-features/&do=findComment&comment=3434082](url)

> While writing this post, I figured out what could be the culprit: the inclination of all bodies seems to be very high, around 23°. I'm pretty sure that this is done in principia because rotating the whole solar system is the only way to implement earths axial tilt.

>The inclination is most likely the problem, yes.  I know why they add it, but I wish they wouldn't... it causes issues like what you discovered. The only solution is to create the RSS bodies file like normal and then manually edit the inclinations to be more reasonable in the file.  There's nothing much KSPTOT can do for it.  I know it's a pain, I'm sorry.﻿

I've been looking to real life mission planning software like GMAT to try and get accurate mission planning abilities but the skills to use it properly are beyond me, even if it may be suitable. Even something as simple as a spreadsheet like [this](http://clowder.net/hop/railroad/Hohmann.xls) would do, but one that works for dates prior to yr 2000.

**Comment by nikolain** [comment]
+1. At least it would be awesome to provide "official" documentation hints on the compatible tools, esp the one that can operate in RSS 1950-2000+ or custom systems. If the tool can do reasonable job with multi-body fly-by's, even better.

**Comment by szundi** [comment]
The inclination is there because Earth in Principia correctly has axial tilt against the plane of the planets. That's like even more real than stock KSP physics with RSS.

**Comment by vladtcvs** [comment]
Agree, this feature would be great. I think, it can be implemented by putting "test" vessel on specific orbit in specific time and user can build flight plan. Now you can do same with real vessel, but if launch window opens in hundreds of days, flight plan will contain all this hundreds of days, and it works pretty slow.

**Comment by gsantos9489** [comment]
+1. That'd be great.
